FreeIT/src/lesson_10/LibraryRunner.java

```diff
@@ -0,0 +1,50 @@
+package lesson_10;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.ListIterator;
+
+public class LibraryRunner {
+
+	public static void main(String[] args) {
+
+		Library pushkina = new Library();
+		Book book1 = new Book(123, "Atlant", "Roman");
+		Book book2 = new Book(345, "Puaro", "Detective");
+		Book book3 = new Book(567, "NLO", "Fantasy");
+
+		pushkina.addBook(book1);
+		pushkina.addBook(book2);
+		pushkina.addBook(book3);
+		List<Book> a = pushkina.getAll();
```

Comment:
all code below is extra and should be removed. You create console application for that purposes

Reply:
ok, I just leave this class for my self) sorry